experimental/uefi/loader/src/crosvm.rs

```diff
@@ -14,32 +14,88 @@
 // limitations under the License.
 //
 
-use crate::vmm::{Params, Vmm};
+use crate::{
+    vmm::{Params, Vmm},
+    ReadWrite,
+};
 use anyhow::Result;
 use async_trait::async_trait;
 use command_fds::tokio::CommandFdAsyncExt;
 use log::info;
 use std::{
+    io::{Error, ErrorKind, Read, Write},
     net::Shutdown,
-    os::unix::{
-        io::{AsRawFd, OwnedFd},
-        net::UnixStream,
-    },
+    os::unix::{io::AsRawFd, net::UnixStream},
     process::Stdio,
 };
+use vsock::VsockStream;
+
+/// The guest VM context ID for virtio vsock connecitons.
```

Comment:
connecitons -> connections. I don't think `VS_GUEST_CID` is public, hence // are enough, or not?

Reply:
Done.